### PR TITLE
Since uploader was not present in the imports , the styles were not showin up

### PR DESCRIPTION
**Pull request by siddhant1**
I have verified that this pull request:

*  [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [ ] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

**Comment by catarak** [comment]
what issue is this fixing?

**Comment by siddhant1** [comment]
Go to side panel > add file > the uploader text is not visible

**Comment by siddhant1** [comment]
image
Text present but not visible
image

**Comment by catarak** [comment]
got it! in the future, please open an issue before making a PR.

File: client/styles/abstracts/_variables.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@ $themes: (
   light: (
     logo-color: $p5js-pink,
     primary-text-color: #333,
+    dropper-text-color: #333,
```
Comment (catarak) [inline client/styles/abstracts/_variables.scss]: can you call this variable "dropzone-text-color"?

**Comment by siddhant1** [comment]
Also I was looking at #709 , I think this pr should address that issue too

File: client/styles/abstracts/_variables.scss [hunk 2]
```diff
@@ -65,6 +66,7 @@ $themes: (
   dark: (
     logo-color: $p5js-pink,
     primary-text-color: $white,
+    dropzone-text-color: $p5js-pink,
```
Comment (catarak) [inline client/styles/abstracts/_variables.scss]: i don't think this is the right color. it should be $black.

**Comment by siddhant1** [comment]
https://github.com/processing/p5.js-web-editor/pull/946#issuecomment-475710056

I think this pr should also tackle this , can you suggest Color changes in dropzone(high contrast)

**Comment by catarak** [comment]
> [#946 (comment)](https://github.com/processing/p5.js-web-editor/pull/946#issuecomment-475710056)
> 
> I think this pr should also tackle this , can you suggest Color changes in dropzone(high contrast)

it should be `$black`. also, it's okay if this doesn't fix #709, but partially fix it.

File: client/styles/abstracts/_variables.scss [hunk 3]
```diff
@@ -112,6 +114,7 @@ $themes: (
   contrast: (
     logo-color: $yellow,
     primary-text-color: $yellow,
+    dropzone-text-color: $yellow,
```
[the remark this answers is not in the document]
Comment (siddhant1) [reply client/styles/abstracts/_variables.scss]: Oh okay!

**Comment by catarak** [comment]
thanks! this is working for me.

**Pull request merged**